### PR TITLE
Update the playground to the latest bdk-cli 

### PR DESCRIPTION
**Pull request by danielabrozzoni**
This needs https://github.com/bitcoindevkit/bdk-cli/pull/116 & https://github.com/bitcoindevkit/bdk-cli/pull/117 to be merged first

Everything under `docs/.vuepress` is auto-generated, so you don't have to review it :)

### CODE REVIEW

File: playground/README.md [hunk 1]
```diff
@@ -1,69 +1,42 @@
...
+cd bdk-cli
+cargo build --no-default-features --features esplora-reqwest,async-interface,compiler --target wasm32-unknown-unknown
 ```
 
-### 🔬 Test in Headless Browsers with `wasm-pack test`
+From inside the `bdk-cli` directory, run `wasm-bindgen`
 
 ```
-wasm-pack test --headless --firefox
+wasm-bindgen --target bundler --out-dir target/wasm32-unknown-unknown/debug/ target/wasm32-unknown-unknown/debug/bdk-cli.wasm
```
Comment (afilini) [inline playground/README.md]: You should make a release build, for size and performance reasons

**Comment by afilini** [comment]
Also I think currently your submodule is pointed to a local path, it should be changed to the https git url

**Comment by danielabrozzoni** [comment]
> Also I think currently your submodule is pointed to a local path, it should be changed to the https git url

No, it's a git submodule, so the local path should be ok

File: playground/README.md (hunk 1 above)
Comment (danielabrozzoni) [reply playground/README.md]: I think it's already a release build, as there's no `--debug`: https://rustwasm.github.io/wasm-bindgen/reference/cli.html

Comment (afilini) [reply playground/README.md]: If cargo is placing the wasm file in target/debug then it's probably not a release build

Comment (danielabrozzoni) [reply playground/README.md]: Oh, I thought you meant the wasm-bindgen part. Updated and re-uploaded the autogenerated files, thanks!

**Review by afilini — APPROVED** [review]
ACK 8671c03d36793cfe3bfb39c83ed3894129c3feca

**Comment by afilini** [comment]
Actually the wasm file is like 25mb, was it built in release mode?

**Comment by danielabrozzoni** [comment]
Sorry, it's 6MB now

**Review by afilini — APPROVED** [review]
ACK c3c4ddc92e67babe0951fc4eea3911ea7635e47d

**Pull request merged**